code/datums/components/defibrillator.dm

```diff
@@ -24,7 +24,20 @@
 	var/emag_proof
 	/// uid to an item that should be making noise and handling things that our direct parent shouldn't be concerned with.
 	var/actual_unit_uid
-
+	/// The time it takes between clicking on the patient and the zap being delivered.
+	var/defib_time = 5 SECONDS
```

Comment:
this should probably be a define, so that admins dont mess with it

Reply:
T R U E